README.md

```diff
@@ -403,6 +403,23 @@ __Used in:__ `SITE CHMOD`
 Returns a unique file name to write to. Client requested filename available if you want to base your function on it. 
 __Used in:__ `STOU`
 
+#### [`renewTlsOptions(tlsOptions)`](src/fs.js#L172)
```

Comment:
@matt-forster 
That makes sense.  I have moved it now to a separate section, and made two other changes:
1. The section header is now no link (to the source) anymore, because all other header titles were also no links.  And otherwise it looked very weird...
2. The link pointed to the wrong source file.